packages/oi4-oec-service-node/src/application/OI4ApplicationFactory.ts

```diff
@@ -61,20 +60,23 @@ export class OI4ApplicationFactory implements IOI4MessageBusFactory {
 
     initialize() {
         const brokerConfiguration: BrokerConfiguration = JSON.parse(readFileSync(this.settingsPaths.mqttSettings.brokerConfig, 'utf8'));
+        const maximumPacketSize = OI4ApplicationFactory.getMaxPacketSize(brokerConfiguration);
         const mqttSettings: MqttSettings = {
             clientId: os.hostname(),
             host: brokerConfiguration.address,
             port: brokerConfiguration.secure_port,
             protocol: MQTTS,
             properties: {
-                maximumPacketSize: OI4ApplicationFactory.getMaxPacketSize(brokerConfiguration)
+                maximumPacketSize: maximumPacketSize
             }
         }
+
+        const opcUaBuilder = new OPCUABuilder(this.resources.oi4Id, this.serviceType, maximumPacketSize);
```

Comment:
The instantiation with new will cause a tight coupling and makes it impossible to use dependency injection (inversion of control)
```suggestion
if(this.opcUaBuilder === undefined){
       this.opcUaBuilder = new OPCUABuilder(this.resources.oi4Id, this.serviceType, maximumPacketSize);
}
```

Reply:
Updated source code according the proposed change.